client/client.go

```diff
@@ -87,6 +87,11 @@ type doer interface {
 	Do(*http.Request) (*http.Response, error)
 }
 
+type ClientGetter interface {
```

Comment:
Normally in Go you'd define the interface type only where it's needed/consumed, for example in `internals/cli/cli.go`.